### DISCUSSION TITLE
task 1 lab 02.03 Pandas

### DISCUSSION THREAD

**Discussion by Jhurtado99** [rramosp/ai4eng.v1#45] [02 - PYTHON]
Cordial saludo profe, estoy tratando de probar mi solución de la tarea 1 del laboratorio de Pandas pero recibo el siguiente error
![image](https://user-images.githubusercontent.com/87033456/124640202-b4fb8880-de52-11eb-87a0-d33ce5dbe193.png)

Realmente no sé de que se trata, podría orientarme en la solución del problema, muchas gracias

**Comment by Felipe-RA** [comment]
Saludos Jhurtado

Este error ocurre porque el **grader** está aplicandole a tu respuesta (lease la variable **answer**) la función `**len()**` para compararla con la respuesta esperable.

Sin embargo, esta variable answer es de tipo `NoneType` , lo que significa que no se le asignó un tipo.

El problema probablemente está en el retorno de tu código
